website/docs/integrations/github-actions/atmos-terraform-plan.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Atmos Terraform Plan
+sidebar_position: 40
+sidebar_label: Terraform Plan
+---
+
+The Cloud Posse GitHub Action for "Atmos Terraform Plan" runs Terraform entirely from GitHub Action workflows.
+
+Given any component and stack in an Atmos supported infrastructure environment, [`github-action-atmos-terraform-plan`](https://github.com/cloudposse/github-action-atmos-terraform-plan) will run `atmos terraform plan`, generate a Terraform [planfile](https://developer.hashicorp.com/terraform/tutorials/automation/automate-terraform), store this planfile in an S3 Bucket with metadata in DynamodDB, and finally format the Terraform Plan result as part of a [GitHub Workflow Job Summary](https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/).
+
+This action is intended to be used with [Atmos Terraform Apply](/integrations/github-actions/atmos-terraform-apply)
+
+## Features
+
+This GitHub Action incorporates superior GitOps support for Terraform by utilizing the capabilities of Atmos, enabling efficient management of large enterprise-scale environments.
+
+* **Implements Native GitOps** with Atmos and Terraform
+* **No hardcoded credentials.** Use GitHub OIDC to assume roles.
+* **Compatible with GitHub Cloud & Self-hosted Runners** for maximum flexibility. 
+* **Beautiful Job Summaries** don't clutter up pull requests with noisy GitHub comments
+* **100% Open Source with Permissive APACHE2 License** means you have no expensive subscriptions or long-term commitments.
+
+![Example Image](/img/github-actions/tf_plan.png)
+
+![Example Image Expanded](/img/github-actions/tf_plan_expanded.png)
+
+## Usage Example
+
```

Comment:
In this example, the action is triggered when certain events occur, such as a manual workflow dispatch or the opening, synchronization, or reopening of a pull request, specifically on the main branch. It specifies specific permissions related to assuming roles in AWS. Within the "plan" job, the "component" and "stack" are hardcoded. In practice, these are usually derived from another action.